search/search_repository.go

```diff
@@ -524,8 +553,39 @@ func (q Query) generateExpression() (criteria.Expression, error) {
 	return res, nil
 }
 
+func extractQueryOptions(q Query) *QueryOptions {
+	qo := QueryOptions{}
```

Comment:
Should this function care about returning an error if it occurs because of let's say bad parameter ?

Reply:
There is always a default value and so an error value is not necessary.